src/signals/incident/components/IncidentForm/index.tsx

```diff
@@ -161,6 +171,9 @@ const IncidentForm = forwardRef<any, any>(
             setIncident(formAction)
             setSubmitting(false)
             next()
+            if (index < steps.length) {
```

Comment:
not anymore the same loading state, but still raising the stepsCompletedCount by one when submitting and the form in valid.